sites/next.skeleton.dev/src/content/docs/installation/nextjs.mdx

```diff
@@ -0,0 +1,68 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+title: Next.js
+description: Install and conifigure Next.js.
+orderBy: 0
+---
+
+export const components = componentSet;
+
+import Process from '@components/docs/Process.astro';
+import ProcessStep from '@components/docs/ProcessStep.astro';
+
+{/* --- */}
+
+<Process>
+    <ProcessStep step="1">
+        ## Create a Project
+        Start by creating a new project using `create-next-app`:
+        ```console
+        npx create-next-app@latest my-app --typescript --tailwind --eslint
+        ```
+    </ProcessStep>
+    <ProcessStep step="2">
+        ## Install Skeleton
+        Install the Skeleton core and Skeleton React packages.
+        ```console
+        npm i -D @skeletonlabs/skeleton @skeletonlabs/skeleton-react
```

Comment:
Do we want to append an ``@next`` tag to this for now or is it just intended as an "installation will work like this but currently doesn't" page?

Reply:
Good call. I'll add this.